File: app/src/main/java/com/gustu/logbook/fragment/home/interfaces/HomeView.java
package com.gustu.logbook.fragment.home.interfaces;

import android.view.View;

import com.gustu.logbook.fragment.home.model.DataItem;
import com.gustu.logbook.fragment.home.model.LogbookResponse;

import java.util.List;

public interface HomeView {
    View _onDataLoad(List<DataItem> dataItemList);
    View _onDataFail();
}

File: app/src/main/java/com/gustu/logbook/fragment/home/interfaces/HomeView.kt
package com.gustu.logbook.fragment.home.interfaces

import android.view.View
import com.gustu.logbook.fragment.home.model.GetLogBook

interface HomeView {
    fun _onDataLoad(dataItemList: List<GetLogBook>?): View
    fun _onDataFail(): View
}
